hamocc/mo_param_bgc.F90

```diff
@@ -177,6 +181,26 @@ module mo_param_bgc
   real, parameter :: rnh4dnrai     = 1./rnh4dnra     ! inverse
   real, parameter :: rnm1          = rnit - 1.
 
+  ! Terrestrial dissolved organic carbon tDOC (river2oceanmip) 
+  ! Low-carbon tDOC
+  real, parameter :: rcar_tdoclc  = 276.                             ! mol C per mol P
+  real, parameter :: rnit_tdoclc  = 25.                              ! mol N per mol P
+  real, parameter :: rhyd_tdoclc  = 2.*rcar_tdoclc+3.*rnit_tdoclc+3. ! mol H per mol P in low-C tDOC
+  real, parameter :: roxy_tdoclc  = rcar_tdoclc + 4.                 ! mol O per mol P in low-C tDOC
+  real, parameter :: ro2ut_tdoclc = (4.*rcar_tdoclc+rhyd_tdoclc-2.*roxy_tdoclc+5.*rnit_tdoclc+5.)  &
+                                  & /4. ! Oxygen utilization per mol tDOClc during remineralisation
+  real, parameter :: ro2utammo_tdoclc = (4.*rcar_tdoclc+rhyd_tdoclc-2.*roxy_tdoclc-3               &
```

Comment:
-3. (and potentially consider a different line break for readability - i.e. keeping `3.*rnit_tdoclc` together)

Reply:
Maybe add the result value in the comment part ...